### PR TITLE
Updating `addons.apt` node to reflect the current nested setup. For #58

### PR DESCRIPTION
**Pull request by solarce**
Updating `addons.apt` node to reflect the current nested setup. For #58

### CODE REVIEW

**Comment by dragon788** [comment]
@solarce Could you try merging recent master changes to this branch? I just ran into the apt addon failure trying to use the linter against a file when building a new project that I want to test offline first.

**Comment by Lin-Buo-Ren** [comment]
Any update for this PR?

**Comment by fingolfin** [comment]
This PR fixes a ton of bugs -- the Travis linter currently even rejects several examples in the Travis docs, and this has been so for years :-(.

**Comment by szepeviktor** [comment]
@BanzaiMan Could you merge this one?

**Comment by onomojo** [comment]
What's the hold up on this? Its been nearly 3 years now. Maybe its time to go back to Jenkins. At least we aren't stuck waiting for years for a simple PR to get merged in to fix the broken linter.

**Comment by onomojo** [comment]
Should this repos just be labeled deprecated? No one appears to actually be actually actively maintaining this thing anymore.

**Comment by szepeviktor** [comment]
TravisCI runs on this!

**Comment by onomojo** [comment]
How could they possibly use this when perfectly executable travis yaml fails their own linter?

**Comment by jyn514** [comment]
Looks like build for linter is failing, any ideas on how to fix? [64-65](https://github.com/travis-ci/travis-yaml/pull/65/files#diff-313d972c99af514b37d506bf305681bbR63) are the problem as per https://travis-ci.org/travis-ci/travis-yaml/jobs/67432591

File: spec/nodes/addons_spec.rb [hunk 1]
```diff
@@ -60,9 +60,11 @@ def addons(input)
       example { expect(addons(ssh_known_hosts: ['git.example.org', 'git.example.com']).ssh_known_hosts).to be == ['git.example.org', 'git.example.com'] }
     end
 
-    context 'apt_packages' do
-      example { expect(addons(apt_packages: 'curl').apt_packages).to be == ['curl'] }
-      example { expect(addons(apt_packages: ['curl', 'git']).apt_packages).to be == ['curl', 'git'] }
+    context 'apt' do
+      example { expect(addons(apt: { 'sources': 'deadsnakes'}).apt.sources).to be == ['deadsnakes'] }
```
Comment (jyn514) [inline spec/nodes/addons_spec.rb]: [This fails the build](https://travis-ci.org/travis-ci/travis-yaml/jobs/67432591).

Comment (onomojo) [reply spec/nodes/addons_spec.rb]: Old Ruby version running on travis maybe? Looks like its failing on the newer : syntax. I'd say either upgrade the ruby version used on travis or change the hashes to the old rocket style 'something' => 'deadsnakes'.

**Pull request closed without merging**